src/components/BrowseTopicsModal/connector.js

```diff
@@ -0,0 +1,32 @@
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import { getFollowedTags } from '../../models/community'
+import { connectedListProps } from '../../util/caching'
+import { fetchTags, followTag } from '../../actions'
+
+const subject = 'community'
+
+export function mapStateToProps (state, { community }) {
+  return {
+    ...connectedListProps(state, {subject, id: community.slug}, 'tags'),
+    followedTags: getFollowedTags(community, state)
+  }
+}
+
+export function mapDispatchToProps (dispatch, { onboarding }) {
+  return {
+    actions: bindActionCreators({
+      fetchTags: (community, offset = 0) =>
```

Comment:
On further thought, why not just modify the fetchTags action to accept a community key (or post, etc) instead of a subject+id (or creating a new fetchTagByCommunity action). I'm seeing that we use fetchTags in only 2 other components... Either way, I think we should push the intention of simplifying our consumption of fetchTags either into that action or a new action so that the "problem" gets presented at the right layer to now or later solve it at.